### ISSUE TITLE
shortcuts twice

### ISSUE THREAD

**Issue by efarcy** [hluk/CopyQ#2085]
Before creating a new issue, see list of [known
issues](https://copyq.readthedocs.io/en/latest/known-issues.html).

**Describe the bug**
When I use copyQ, I must press twice the shortcut to paste in a window.

**To Reproduce**
1. Go to a text editor
2. Click on copy a text in the clipboard
3. Press enter to creat a new line
4. Try to paste the text with the shortcut
5. You must press a second time the shortcut

**Expected behavior**
Just one time for the shortcut
**Screenshots**

**Version, OS and Environment**
(Get details from `copyq version` command if possible.)
 - Application Version [e.g. 3.7.2] 3.4.5
 - OS [e.g. Windows] linux mint
 - Desktop environment, window manager (if applicable) cinnamon

**Additional context**

**Comment by hluk** [comment]
CopyQ by default does not change/override the shortcuts for pasting (Shift+Insert or Ctrl+V) - it does not define any global shortcut by default. Or do you refer to a different shortcut?

**Comment by efarcy** [comment]
I have this in my config :+1: 

![image](https://user-images.githubusercontent.com/83125577/189879161-a7ed4472-72c2-422d-98fc-8c6b3d1db914.png)

When i must press twice meta+alt+V (it's the same problem with meta+alt+C) to open this window : 

![image](https://user-images.githubusercontent.com/83125577/189879638-b47794fb-a0a7-42a8-95b0-a6e98492abb8.png)

**Comment by efarcy** [comment]
One solution I found is to press meta+alt+V+C to open this window.
Previously, it worked well.

**Comment by efarcy** [comment]
it' my version

**Comment by hluk** [comment]
> it' my version

In the description it says version is 3.4.5 - which is actually invalid.

Can you post output of `copyq version`?

**Comment by efarcy** [comment]
sorry, it was a mistake : 
![image](https://user-images.githubusercontent.com/83125577/190116793-a6d9333c-479c-4302-a6d5-4de8b237b326.png)

**Comment by Vishwas-kc** [comment]
Facing the same issue

I am on pop!_os 
CopyQ version 6.3.2~jammy

I added a global shortcut Super+v for "Show the tray menu",
It is displayed as (Meta+v) on the shortcuts screen.

The issue is if I press super+v it displays "v" on the screen on the first press, only on the second press it shows the tray menu.

**Comment by NA0341** [comment]
Did you try different shortcuts?
(other letters, other key than Meta, etc.)

I'm on Manjaro GNOME 44 and `Meta + V` opens the notification panel.
Many times when shortcuts don't work as intended it's because some other Software is interfering, because it listens for shortcuts as well [...].

**Comment by Vishwas-kc** [comment]
Everything other than meta works. But I like to set it to `Meta + V` because I am used to using it on windows.

**Comment by efarcy** [comment]
> The issue is if I press super+v it displays "v" on the screen on the first press, only on the second press it shows the tray menu.

It's  my problem with meta+V, or meta+other key, but it's good with alt+v. 
meta is the problem.

**Comment by NA0341** [comment]
> > The issue is if I press super+v it displays "v" on the screen on the first press, only on the second press it shows the tray menu.
> 
> It's my problem with meta+V, or meta+other key, but it's good with alt+v. meta is the problem.

As I said: it's very likely that this is caused by another program listening for the same shortcuts.

Considering the fact that the `meta`/`super` key is affected, it's very likely that the program interfering is your window manager, composer or some other part of the desktop environment.
Since `meta` is used by most DE's for desktop actions such as workspaces (desktops), window management or other inbuilt features.

In short: find out what is interfering - and either make a workaround or change your setup.

Tip: You can pack yourself some live ISO's on a USB Stick with Ventoy on it - and test with different Desktop environments as well as your current system as a live version.

**Comment by nsbuman** [comment]
There is a file search mechanism in Nemo that is triggered when you start typing and it looks like it ignores the super key and just searches for 'V' on the desktop requiring a second key press.

![Image](https://github.com/user-attachments/assets/a13aada2-7209-4dac-8046-e45fae60d1fe)

**Comment by GarThor** [comment]
Same issue.  Also wanted to set it to `Win+V` (`Super/Meta+V`), because I'm used to the windows shortcut.

I checked the keyboard shortcuts in the settings app, and I don't see anything in there that would be overriding the `Super+V` shortcut.  Is there another place I can check?

Version 7.1.0 if it matters.

**Comment by SoleInvictus** [comment]
Same issue here: using Meta+V (or meta plus any key) requires a second keypress to register. I've been using Diodon, but much prefer CopyQ. I have no issues assigning meta + <any key> to any other shortcuts or programs, so it seems to be particular to CopyQ.

I tried making a custom command that just runs "copyq show", but it brings the window up in focus, so the "hide when not in focus" feature doesn't trigger, plus I prefer it appearing at my cursor, not its last position.

**Comment by Chealer** [comment]
@nsbuman wrote:
> There is a file search mechanism in Nemo that is triggered when you start typing and it looks like it ignores the super key and just searches for 'V' on the desktop requiring a second key press.

What do you mean by "Nemo"? [That file manager](https://en.wikipedia.org/wiki/Nemo_(file_manager))?

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>

**Comment by Chealer** [comment]
@SoleInvictus, @GarThor and @Vishwas-kc: Which desktop environment do you use?

@SoleInvictus: Which version do you use?

**Comment by SoleInvictus** [comment]
@Chealer 

Linux Mint. I unfortunately have since uninstalled CopyQ, so I'm not sure of the version.

**Comment by Chealer** [comment]
@SoleInvictus: Linux Mint is a GNU/Linux distribution, not a [desktop environment](https://en.wikipedia.org/wiki/Desktop_environment).

This looks like a duplicate of ticket #1267.

**Comment by SoleInvictus** [comment]
@Chealer Sorry, Cinnamon! I'm sick with a fever, don't judge too harshly.

**Comment by GarThor** [comment]
> [@SoleInvictus](https://github.com/SoleInvictus), [@GarThor](https://github.com/GarThor) and [@Vishwas-kc](https://github.com/Vishwas-kc): Which desktop environment do you use?
> 
> [@SoleInvictus](https://github.com/SoleInvictus): Which version do you use?

It's been so long since I posted that comment I don't actually remember.

On my desktop, I'm pretty sure it was Ubuntu.  I like KDE, so it might have been KDE... I just upgraded my ubuntu to 25.01 though (a mistake that I need to revert), and it's forced me to use Gnome, because they removed x11 support, and I really like to use Solaar with my logitech gear... unfortunately Solaar only works under wayland with a gnome plugin, and there's no alternative for alternate DE's.

On my laptop I use Mint/XFCE... because the thing is a potato, and I need the perf for it to be anywhere near usable.

I also spent a rather small amount of time on Bazzite this year (just giving it a try)... probably not that though... It kinda sucked... (don't tell the Bazzite people, they will jump down my throat about how good it is).

**Comment by hluk** [comment]
I think the solution could be to use **Portal**. Currently, it is only used on Wayland.

I will make it optional on X11 using an environment variable `COPYQ_USE_PORTAL=1` (#3355).

The feature requires [org.freedesktop.portal.GlobalShortcuts portal](https://flatpak.github.io/xdg-desktop-portal/docs/doc-org.freedesktop.portal.GlobalShortcuts.html) to be available in the desktop environment. It uses some tricks to allow modifying shortcut from the app, but it is still much better than the X11 solution, which needs listening to Xorg events and translating shortcuts keys. 🫣
